src/ui-lib/sass/04-objects/_section.scss

```diff
@@ -1,3 +1,7 @@
-.grav-o-section--centered {
-  text-align: center;
+.grav-o-section {
+  margin-top: $grav-sp-l;
+
+  &--centered {
```

Comment:
As per my comment above, perhaps we can do without the `--centered` modifier here.